### ISSUE TITLE
ugettext_lazy deprecation warnings

### ISSUE THREAD

**Issue by si14** [django-parler/django-parler#277]
When imported in a project run with django 3.0 and `PYTHONDEVMODE=1`, django-parler generates the following warning:

```
site-packages/parler/models.py:1001: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy()
```

If my understanding is correct, https://github.com/django-parler/django-parler/pull/275 should fix this.

**Comment by xi** [comment]
This should be fixed in #275.

**Comment by xi** [comment]
#275 has been merged and released as part of [2.2](https://github.com/django-parler/django-parler/blob/master/CHANGES.rst#changes-in-22-2020-09-06). Can this be closed?

**Comment by si14** [comment]
Yes, sorry!
